<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Azo Sans Medium</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>bankSimulator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220);
color: black;</string>
    </property>
    <property name="text">
     <string>Upload Design</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(220, 220, 220);
color: black;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="selected_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>53</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0)</string>
    </property>
    <property name="text">
     <string>0 Selected.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="placeholderText">
     <string>Theme: (e.g. Valentines or Christmas)</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="shopname_label">
       <property name="styleSheet">
        <string notr="true">color: white;
</string>
       </property>
       <property name="text">
        <string>Shop Names:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="description_label">
       <property name="styleSheet">
        <string notr="true">color: white;
</string>
       </property>
       <property name="text">
        <string>Descriptions:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="shopname_list">
       <property name="styleSheet">
        <string notr="true">color: white;
</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="toggle_button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(220, 220, 220);
color: black;

</string>
       </property>
       <property name="text">
        <string>Toggle All</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QPlainTextEdit" name="description">
       <property name="styleSheet">
        <string notr="true">color: white;
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>371</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: gray;
color: black;</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="action_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="terminal">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
